{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/NoSuchStream\/location"},"sections":[],"metadata":{"role":"symbol","title":"location","roleHeading":"Instance Property","externalID":"s:8NIOHTTP20A6ErrorsO12NoSuchStreamV8locationSSvp","symbolKind":"property","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"modules":[{"name":"NIOHTTP2"}]},"abstract":[{"text":"The location where the error was thrown.","type":"text"}],"variants":[{"paths":["\/documentation\/niohttp2\/niohttp2errors\/nosuchstream\/location"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/NoSuchStream"]]},"primaryContentSections":[{"declarations":[{"platforms":["Linux"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"location"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}]}],"kind":"declarations"}],"kind":"symbol","references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/NoSuchStream":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/NoSuchStream","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"NoSuchStream","kind":"identifier"}],"navigatorTitle":[{"text":"NoSuchStream","kind":"identifier"}],"type":"topic","kind":"symbol","title":"NIOHTTP2Errors.NoSuchStream","abstract":[{"type":"text","text":"An attempt was made to issue a write on a stream that does not exist."}],"url":"\/documentation\/niohttp2\/niohttp2errors\/nosuchstream","role":"symbol"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"type":"topic","url":"\/documentation\/niohttp2","kind":"symbol","title":"NIOHTTP2","abstract":[],"role":"collection","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/NoSuchStream/location":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/NoSuchStream\/location","abstract":[{"text":"The location where the error was thrown.","type":"text"}],"type":"topic","title":"location","role":"symbol","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"location"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/nosuchstream\/location","kind":"symbol"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"abstract":[{"type":"text","text":"Errors that NIO raises when handling HTTP\/2 connections."}],"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NIOHTTP2Errors"}],"role":"symbol","type":"topic","url":"\/documentation\/niohttp2\/niohttp2errors","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"NIOHTTP2Errors"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","title":"NIOHTTP2Errors"}}}